src/main/helpers/formatters.ts

```diff
@@ -47,6 +47,7 @@ export const xatabFormatter = (title: string) =>
     .replace(/(v\.?([0-9]| )+)+([0-9]|\.|-|_|\/|[a-zA-Z]| )+/, "");
 
 export const tinyRepacksFormatter = (title: string) => title;
+export const onlinefixFormatter = (title: string) => title;
```

Comment:
Please add a proper formatter for the repacker if necessary. Any repacker that post their games with anything that goes beyond the game title should have its own RegExp that will force the title to be formatted like a regular game title. You can check FitGirl's one as an example.